models/Translation/Dao.php

```diff
@@ -168,6 +168,11 @@ public function getAvailableDomains(): array
     public function isAValidDomain(string $domain): bool
     {
         try {
+            $translationDomains = $this->model->getRegisteredDomains();
+            if (!in_arrayi($domain, $translationDomains)) {
+                return false;
+            }
+
             $this->db->fetchOne(sprintf('SELECT * FROM translations_%s LIMIT 1;', $domain));
```

Comment:
What about adding this?
```suggestion
            $this->db->fetchAssociative(sprintf('SELECT %s, %s, %s, %s FROM translations_%s LIMIT 1',
                $this->db->quoteIdentifier('key'),
                $this->db->quoteIdentifier('language'),
                $this->db->quoteIdentifier('text'),
                $this->db->quoteIdentifier('type'),
                $domain
            ));
```
Similarly to https://github.com/pimcore/pimcore/pull/14087/files, this checks if the table structure is fine to be considered a valid translation domain.

Reply:
this is the main reason for adding the config so shouldn't be necessary to check structure.

Reply:
ah right, so it's up to the devs to make sure it's well configured